main/guides/getting-started/swaparoo-how-to-pay-somone-explainer.md

```diff
@@ -0,0 +1,5 @@
+# Swaparoo Contract
+
+## How to Pay Someone
```

Comment:
This video is an interesting walk-thru of the swap contract, so it might be a good way to address...

 - #940

```suggestion
## How to Swap Assets
```

But most of it is not relevant to "how to pay someone" (#941). The how-to-pay someone part of the contract is mainly:

```js
    const secondDepositFacet = await E(depositFacetFromAddr).lookup(
      secondPartyAddress,
      'depositFacet',
    );

    await E(secondDepositFacet).receive(secondSeatInvitation);
```

https://github.com/Agoric/dapp-agoric-basics/blob/1b7c376328fb2c3cb6a50a32b645c0a0b539732c/contract/src/swaparoo.contract.js#L167-L172